docs/icd.md

```diff
@@ -0,0 +1,66 @@
+# Interface Control Document (ICD) - `svc-pricing`
+
+<center>
+
+<img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
+
+</center>
+
+## Overview
+
+This document details the common software interfaces required by all services in the Arrow ecosystem.
+
+Each service may add additional interfaces.
+
+Attribute | Description
+--- | ---
+Status | Draft
+
+## Related Documents
+
+Document | Description
+--- | ---
+:construction: Requirements & User Stories :construction: | Requirements and user stories
+
+## Frameworks
+
+See the Services ICD.
+
+## REST
```

Comment:
You can remove the REST section, I don't believe pricing will have a REST interface outside of the common one, which is defined in the high-level ICD

Reply:
ICD related comments will be addressed in a dedicated ICD PR.